src/store/index.js

```diff
@@ -546,6 +547,11 @@ const store = new Vuex.Store({
       Vue.set(state, 'recent', arr)
     },
 
+    SET_DARK_THEME (state, bool) {
+      localStorage.setItem('darkTheme', bool)
+      Vue.set(state, 'darkTheme', bool)
```

Comment:
For boolean values, you can just do `state.darkTheme = bool`. The reason we use `Vue.set` on object and array values is that it's harder for Vue's reactivity to detect changes on those values. However, for "simple" values like booleans, numbers and strings where we're always overwriting the value, we can just directly set it. See: https://vuejs.org/v2/guide/reactivity.html#Change-Detection-Caveats

Reply:
oh cool thats good to know 👍